src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -99,6 +99,10 @@ public partial class UpdateDurableFunction
                     instanceWatermarks);
             }
 
+            if (instanceWatermarks.Count > 0)
+            {
+                _updateMeter.UpdatedInstances.Add(instanceWatermarks.Count);
```

Comment:
So, where did we land on addressing possible duplicates?

Reply:
We will be addressing the duplicates as part of a different work item.